flashinfer/gdn_kernels/gdn_decode_bf16_state.py

```
@@ -2044,169 +2094,817 @@ def run_gdn_decode_bf16state_mtp(
+        h0_source.layout.shape[1],
+        h0_source.layout.shape[2],
+    )
+
+    num_v_tiles = cute.ceil_div(v_dim, tile_v)
+    grid_size = B * HV * num_v_tiles
+
+    # SMEM: for T>1 include shared sQ/sK (1 copy) + sGB; T=1 needs minimal
+    smem_bytes = 128  # alignment padding
+    if T > 1:
+        smem_bytes = (
+            4 * T * (K + 8)  # sQ: T × (K+8) × 4 bytes (shared, one copy)
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)